packages/lab/src/list-next/ListItemNext.css

```diff
@@ -0,0 +1,71 @@
+/* Default variables applied to the root element */
+.saltListItemNext {
+  /* Color */
+  --list-item-text-color: var(--salt-text-primary-foreground);
+  --list-item-height: var(--salt-size-stackable);
+  --list-item-background: none;
+}
+
+/* Style applied to the root element */
+.saltListItemNext {
+  color: var(--list-item-text-color);
+  background: var(--list-item-background);
+  height: var(--list-item-height, auto);
+  padding-inline: var(--salt-size-unit);
+  display: flex;
+  gap: var(--salt-size-unit);
+  position: relative;
+  align-items: center;
+  white-space: nowrap;
+  transition: background var(--salt-duration-perceptible) ease-in-out;
+}
+
+.saltListItemNext > * {
+  pointer-events: none;
+}
+
+/* Style applied to presentation items (headers and empty message) */
+.saltListItemNext[role="presentation"] {
+  cursor: default;
+}
+
+/* Pseudo-class applied to the root element on focus */
+.saltListItemNext-focused {
+  outline: var(--salt-focused-outline);
+  outline-offset: calc(var(--salt-focused-outlineWidth) * -1);
+}
+
+/* active list item when focused */
+.saltListItemNext:hover:not([role="presentation"]):not(.saltListItemNext-disabled),
+.saltListItemNext-focused {
+  --list-item-background: var(--salt-selectable-background-hover);
+  cursor: var(--salt-selectable-cursor-hover);
+}
+
+.saltListItemNext[aria-selected="true"] {
+  border-block: var(--salt-size-border) var(--salt-container-borderStyle) var(--salt-selectable-borderColor-selected);
+  --list-item-background: var(--salt-selectable-background-selected);
+  --saltIcon-color: var(--salt-selectable-foreground-selected);
+}
+
+/* Style applied to disabled items */
+.saltListItemNext-disabled {
+  --list-item-text-color: var(--salt-text-primary-foreground-disabled);
+  cursor: var(--salt-selectable-cursor-disabled);
+}
+.saltListItemNext-disabled .saltText {
+  color: var(--list-item-text-color);
+}
+
+/* disabled list item when selected */
+.saltListItemNext-disabled[aria-selected="true"] {
+  border-block: var(--salt-size-border) var(--salt-container-borderStyle) var(--salt-selectable-borderColor-selectedDisabled);
+  --list-item-background: var(--salt-selectable-background-selectedDisabled);
```

Comment:
we usually take Button.css as the standard, so having e.g. --list-item-background and --list-item-background-disabled in the root then setting disabled: var(--list-item-background-disabled) here